ci/test_examples.sh

```diff
@@ -1,3 +1,3 @@
 for example in $(find ./examples/ -iname *.py); do
-  python $example
+  CI_MODE=True python $example
```

Comment:
```suggestion
  CI_MODE=1 python $example
```

Reply:
Is there a difference compared to the notebook job that we merged?

https://github.com/pydata/sparse/blob/0e95f8b5c6e98f73bfb2c5faba71c4f90abc17af/ci/test_notebooks.sh#L1

We use `True` there. Or do we want to also change it here?

Reply:
I'd say also change it there. If there's repetition we should factor it out.

Reply:
Done!